src/Tools/Microsoft.CodeAnalysis.Toolset.Open/Targets/VSL.Settings.targets

```diff
@@ -58,17 +58,27 @@
     <RoslynDiagnosticsPropsFilePath>$(NuGetPackagesPath)\Microsoft.Net.RoslynDiagnostics.1.0.0-rc2-20150313-01\build\Microsoft.Net.RoslynDiagnostics.props</RoslynDiagnosticsPropsFilePath>
     <AdditionalFileItemNames>$(AdditionalFileItemNames);PublicAPI</AdditionalFileItemNames>
   </PropertyGroup>
-  <ImportGroup Label="GlobalNuGets">
+
+  <!-- Import the props files from the toolset NuGet packages if we're not in a bootstrap build -->
+  <ImportGroup Label="GlobalNuGets" Condition="'$(BootstrapBuild)' == ''">
     <Import Project="$(ToolsetCompilerPropsFilePath)"
             Condition="Exists('$(ToolsetCompilerPropsFilePath)')" />
     <Import Project="$(RoslynDiagnosticsPropsFilePath)"
             Condition="Exists('$(RoslynDiagnosticsPropsFilePath)') And ('$(UseRoslynAnalyzers)' == 'true')" />
   </ImportGroup>
-  <Target Name="EnsureGlobalNuGets">
+
+  <Target Name="EnsureGlobalNuGets" Condition="'$(BootstrapBuild)' == ''">
     <Error Condition="!$(CscToolPath.Contains('$(ToolsetCompilerPackageName)'))"
```

Comment:
This needs to be updated to look for the FakeSign package as well -- it needs to error even if doing a BootstrapBuild, no?